src/config.js

```diff
@@ -4,7 +4,7 @@
 module.exports = {
   name: require('../package').name,
   postgres: {
-    mock: process.env.MOCK,
+    mock: process.env.MOCK && process.env.MOCK !== 'false',
```

Comment:
Perhaps we should follow a similar approach to the one we used in lev-api: https://github.com/UKHomeOffice/lev-api/blob/master/config.js#L15

Reply:
Fixed